src/ucp/core/ucp_context.c

```diff
@@ -821,6 +826,12 @@ ucs_status_t ucp_init_version(unsigned api_major_version, unsigned api_minor_ver
     ucs_queue_head_init(&context->tag.expected);
     ucs_queue_head_init(&context->tag.unexpected);
 
+    if (context->config.features & UCP_FEATURE_FAULT_TOLERANCE) {
+        ucp_tag_eager_sync_proto_p = &ucp_tag_eager_sync_unsupport_proto;
```

Comment:
would be simple to check this in ucp_tag_send_sync_nb - since it's not fast-path call (need ack from remote) one branch should not be a problem